challenges/funding.md

```diff
@@ -0,0 +1,108 @@
+# Funding
+
+Funding RSE can be challenging as we are often split between different forms of funding
+
+## Glossary
+
+- **Directly Incurred (DI)**: Costs that are directly attributable to a specific
+  project or activity.
+- **Directly Allocated (DA)**: Costs that are shared across multiple projects or
+  activities.
+- **Overheads**: Costs that are not directly attributable to a specific project
+  or activity.
+- **Full Economic Cost (FEC)**: The total cost of a project, including Directly
+  Incurred, Directly Allocated, and Overheads.
+- **Cost Recovery**: The process of reclaiming costs from a project or activity.
+
+## Funding Models
+
+### TRAC
+
+The Transparent Approach to Costing (TRAC) is a methodology used by UK higher
+education institutions to calculate the Full Economic Cost (FEC) of research
+projects. The FEC includes Directly Incurred (DI), Directly Allocated (DA), and
+Overheads. Using the TRAC approach means that a day rate is calculated for each
+RSE, which is then used to calculate the cost of the RSE's time on a project. This
+rate is based on FEC and is used to recover costs from projects.
+
+**NOTE**: Applications for TRAC are currently blocked
+
+Timesheets are required and then billed in arrears.
+
+### Cost Recovery
+
+Some RSE groups are underwritten by the institution and must cost recover from projects.
+
+Cost recovery targets vary between institutions and can be as high as 100%.
+On average this target is ??
+
+### DI funded
+
+- When charging as a DI cost, the RSE is directly funded by the project. We have
+  to be careful that there is no double counting of costs as funders may have already
+  paid for the overheads indirectly.
+- Overheads generally passed to funder.
+
+### DA funded
+
+- When charging as a DA cost, the RSE is funded by the institution and the costs
+  are shared across multiple projects.
+- Overheads generally not passed to funder
+
+### As a post doc
+
+Some RSEs are funded as post docs.
```

Comment:
```suggestion
Some RSEs are funded as post docs, and therefore are a DI cost.
```

Reply:
Some RSEs are seen as post-docs and therefore DA cost. Members of my group specifically are added as "Researcher DA" to RGs.

Reply:
Isn't it lovely how inconsistent different institutions can be :)

In my experience, typically DA means you're either on a permanent contract with a small approximate slice of your time on a project (like an academic project lead) or part of a staff pool (like many RSEs) but not an individual on a fixed term contract (like most post docs).